worker/buildbot_worker/commands/fs.py

```diff
@@ -261,3 +261,23 @@ def start(self):
             self.sendStatus(
                 {'header': '%s: %s: %s' % (self.header, e.strerror, dirname)})
             self.sendStatus({'rc': e.errno})
+
+
+class RemoveFile(base.Command):
```

Comment:
arg.. its annoying to require a change in the slave.
This means that people will need to upgrade.

We at least need to:
- increment the worker version
- if the worker version is < 3.1, then we should use another method. probably overwrite the file with empty string, or use rmdir command (which looks like baked by rm -rf on unix)
.